requirements/_base.txt

```diff
@@ -13,6 +13,6 @@ python-ooxml
 django-rest-swagger==2.1.0
 djangorestframework==3.5.3
 Markdown==2.6.7
-django-filter==1.0.0
+django-filter==1.1.0
 Ebooklib==0.16
```

Comment:
This needs bumping to:
Ebooklib==0.17
to merge with https://github.com/booktype/Booktype/pull/895